modules/prepare-ocp-for-bare-metal-builds.adoc

```diff
@@ -0,0 +1,199 @@
+* You are logged into {ocp} as a cluster administrator.
+
+.Procedure 
+
+. Enter the following command to create a project where Builds will be run, for example, `bare-metal-builder`:
++
+[source,terminal]
+----
+$ oc new-project bare-metal-builder
+----
+
+. Create a new `ServiceAccount` in the the `bare-metal-builder` namespace by entering the following command:
++
+[source,terminal]
+----
+$ oc create sa -n bare-metal-builder quay-builder
+----
+
+. Enter the following command to grant a user the `edit` role within the `bare-metal-builder` namespace:
++
+[source,terminal]
+----
+$ oc policy add-role-to-user -n bare-metal-builder edit system:serviceaccount:builder:quay-builder
+----
+
+. Enter the following command to retrieve a token associated with the `quay-builder` service account in the `bare-metal-builder` namespace. This token is used to authenticate and interact with the {ocp} cluster's API server.
++
+[source,terminal]
+----
+$ oc sa get-token -n bare-metal-builder quay-builder
+----
+
+. Identify the URL for the {ocp} cluster's API server. This can be found in the {ocp} web console.
+
+. Identify a worker node label to be used when scheduling _build jobs_. Because _build pods_ must run on bare metal worker nodes, typically these are identified with specific labels.
++
+Check with your cluster administrator to determine exactly which node label should be used. 
+
+. Obtain the Kube API Server's certificate authority (CA) to add to {productname}'s extra certificates. 
+
+.. Enter the following command to obtain the name of the secret containing the CA:
++
+[source,terminal]
+----
+$ oc get sa openshift-apiserver-sa --namespace=openshift-apiserver -o json | jq '.secrets[] | select(.name | contains("openshift-apiserver-sa-token"))'.name
+----
+
+.. Obtain the `ca.crt` key value from the secret in the {ocp} Web Console. The value begins with *"-----BEGIN CERTIFICATE-----"`*. 
+
+.. Import the CA to {productname}. Ensure that the name of this file matches the `K8S_API_TLS_CA` field used in Step 9. 
+
+. Create the following `SecurityContextConstraints` resource for the `ServiceAccount`:
++
+[source,yaml]
+----
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  name: quay-builder
+priority: null
+readOnlyRootFilesystem: false
+requiredDropCapabilities: null
+runAsUser:
+  type: RunAsAny
+seLinuxContext:
+  type: RunAsAny
+seccompProfiles:
+- '*'
+supplementalGroups:
+  type: RunAsAny
+volumes:
+- '*'
+allowHostDirVolumePlugin: true
+allowHostIPC: true
+allowHostNetwork: true
+allowHostPID: true
+allowHostPorts: true
+allowPrivilegeEscalation: true
+allowPrivilegedContainer: true
+allowedCapabilities:
+- '*'
+allowedUnsafeSysctls:
+- '*'
+defaultAddCapabilities: null
+fsGroup:
+  type: RunAsAny
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: quay-builder-scc
+  namespace: builder
```

Comment:
Here the namespace should be ` bare-metal-builder`, because we are taking `bare-metal-builder` as example.

Reply:
Fixed!